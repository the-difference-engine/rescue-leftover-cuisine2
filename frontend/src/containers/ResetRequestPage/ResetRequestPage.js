import React from 'react';
import { AuthPage } from '../../components/AuthItems/AuthItems';
import ResetRequest from '../../components/ResetRequest/ResetRequest';

const ResetRequestPage = () => (
  <AuthPage>
    <ResetRequest />
  </AuthPage>
);

export default ResetRequestPage;
