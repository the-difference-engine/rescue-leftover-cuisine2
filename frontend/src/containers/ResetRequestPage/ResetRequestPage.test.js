import React from 'react';
import renderer from 'react-test-renderer';

import ResetRequestPage from './ResetRequestPage';

it('renders correctly', () => {
  const component = renderer
    .create(<ResetRequestPage />)
    .toJSON();

  expect(component).toMatchSnapshot();
});
