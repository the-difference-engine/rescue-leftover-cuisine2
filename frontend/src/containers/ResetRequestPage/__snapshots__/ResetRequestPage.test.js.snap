// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="auth-page container-fluid"
>
  <div
    className="row"
  >
    <div
      className="container-fluid text-center"
    >
      <a
        href="/"
      >
        <img
          alt="RLC Logo"
          className="logoCenter"
          src="RLC_logo.png"
        />
      </a>
    </div>
  </div>
  <div
    className="auth-page-content"
  >
    <div
      className="auth-cards-wrapper d-flex flex-row justify-content-center"
    >
      <div
        className="auth-card"
      >
        <h3
          className="auth-card-header"
        >
          Enter Email to Reset Password
        </h3>
        <form
          className="auth-card-form"
          onSubmit={[Function]}
        >
          <div
            className="form-group row align-middle"
          >
            <input
              className="auth-card-input form-control-lg"
              id="auth-card-input-resetrequest-email"
              maxLength="128"
              minLength="0"
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              pattern="^.+@.+\\\\..+$"
              placeholder="Email"
              required={true}
              type="text"
            />
            <label
              htmlFor="auth-card-input-resetrequest-email"
            >
              Email
            </label>
          </div>
          <div
            className="row form-group"
          >
            <button
              className="auth-card-submit btn btn-lg btn-block"
              onClick={[Function]}
              type="button"
            >
              Reset Password
            </button>
          </div>
        </form>
        <div
          className="auth-card-error row align-middle"
        />
        <div
          className="auth-card-success row align-middle"
        />
      </div>
    </div>
  </div>
  <div
    className="auth-footer row"
  >
    <div
      className="container-fluid containerSection"
    >
      <div
        className="footer"
      >
        <div
          className="row"
        >
          <div
            className="col-md-4"
          />
          <div
            className="col-md-4 company"
          >
            <span>
              Â©2018 Rescuing Leftover Cuisine
            </span>
          </div>
          <div
            className="col-md-4 iconSection"
          >
            <span
              className="float-right"
            >
              <a
                href="https://www.youtube.com/channel/UCJKK9jkEIHt8j17cHPHCPQw"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-youtube fa-2x iconStyle"
                />
              </a>
              <a
                href="https://www.facebook.com/RescuingLeftoverCuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-facebook-f fa-2x iconStyle"
                />
              </a>
              <a
                href="https://twitter.com/rescuingcuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-twitter fa-2x iconStyle"
                />
              </a>
              <a
                href="https://www.instagram.com/rescuingleftovercuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-instagram fa-2x iconStyle"
                />
              </a>
              <a
                href="https://www.linkedin.com/company/rescuing-leftover-cuisine"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fab fa-linkedin-in fa-2x iconStyle"
                />
              </a>
              <a
                href="mailto:info@resuingleftovercuisine.org"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fas fa-envelope fa-2x iconStyle"
                />
              </a>
            </span>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-md-4"
          />
          <div
            className="col-md-4 designBySection"
          >
            <p>
              <span>
                Designed by 
              </span>
              <span
                className="designByCo"
              >
                Fluid Designs. 
              </span>
              App icons by
              <a
                href="https://icons8.com/"
                rel="noopener noreferrer"
                target="_blank"
              >
                 icons8.
              </a>
            </p>
          </div>
          <div
            className="col-md-4"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
